src/templates/home/Home.js

```diff
@@ -56,6 +56,9 @@ const Home = () => {
         if (CheckForm(formData)) {
             setSearching(true);
 
+            formData['interactionType'] = formData['interaction'].substring(0, formData['interaction'].indexOf(","));
```

Comment:
Yes it will be, the API returns the complete type consisting of the interactionType and the interaction (method) -> 
when the user chooses pollinated by it returns: pollinates,pollinatedBy. Seperating by comma gives the interactionType: 'pollinates' as well as the interaction (method): 'pollinatedBy'. The code should in this case work because the API will not format strings with more commas